csvhead.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""
+This scripts allows user to see first n lines in *.csv file.
+"""
+
+from argparse import ArgumentParser
+import sys
+
+DESCRIPTION = 'csvpp - Print header and first lines of input.'
+EXAMPLES = 'example: cat file.csv | csvhead -n 100'
+
+def print_row(row, column_widths, output_stream):
+    """
+    Prints a row in human-readable format taking column widths into account
+    :param row: row represented as a list of columns
+    :param column_widths: a list of column list widths to be used for pretty printing
+    :param output_stream: a stream to pretty print the row
+    """
+    output_line = '|'
+    for i, column in enumerate(row):
+        output_line += ' ' + column + ' ' * (column_widths[i] - len(column) + 1) + '|'
+    output_line += '\n'
+    output_stream.write(output_line)
+
+
+def main():
+    """
+    Reads line per line and writes it in output_stream.
+    """
+    args = parse_args()
+    print(args)
+    input_stream = open(args.file, 'r') if args.file else sys.stdin
+    output_stream = open(args.output_file, 'r') if args.output_file else sys.stdout
+
+    lines = input_stream.readlines()
```

Comment:
This line loads the whole file into memory. Do you really want it? And what if the file size is 10GB? I guess that it still should be possible to implement low-overhead csvhead.